### ISSUE TITLE
[Feature request] Wiki Rework

### ISSUE THREAD

**Issue by Cheezik** [vn7n24fzkq/github-profile-summary-cards#172]
**Context**

Hello, i am creating much needed wiki rework for this repo (it deserves one) 
I am re-styling the whole layout and also updating old/ no longer working code

What are you trying to do and how would you want to do it differently? 

Basically Reworking the entire Github wiki

Can you achieve the same result doing it in an alternative way? Is the alternative considerable?

**Has the feature been requested before?**
No, not to my knowlage.

**If the feature request is approved, would you be willing to submit a PR?**
Yes.

**Comment by vn7n24fzkq** [comment]
Hi, @Cheezik .

Thank you for your efforts in redesigning the Github wiki for this repository. Your work is greatly appreciated. Regarding your proposal, I am fully supportive and believe it is a great plan to improve the quality and relevance of the wiki. Once your feature request is approved, I will review it promptly and provide feedback. Additionally, I'd like to inform you that I am currently on the Lunar New Year holiday, but I will carefully review your proposal upon my return. Thank you for your patience.

**Comment by Cheezik** [comment]
Its all good, i have pretty much finished the rework (link to the fork: https://github.com/Cheezik/github-profile-summary-cards/tree/Wiki-Rework. Branch: Wiki-Rework).

Im planning on adding links under the gifs when i have the time to do so, just let me know anything you'd like to change.

Happy Lunar New year ^^

**Comment by vn7n24fzkq** [comment]
Great! Everything looks fantastic!
Once this rework is finished, I believe we can create a documentation folder for the wiki pages in this repository.
This will allow us to easily manage the wiki page and collaborate by creating new pull requests in the future.
The only thing left for me to do is to write a new GitHub Action CI process to automatically update the wiki Git repository.

This way, your commits and contributions will be visible in this repository. What do you think?

**Comment by Cheezik** [comment]
Sounds perfect, i will try to jump onto final changes asap (possibly tomorrow), Once thats done i think its ready

**Comment by vn7n24fzkq** [comment]
Awesome!

**Comment by Cheezik** [comment]
Alright, sorry for the masive delay, had some urgent things that had to be done. anyways i think that the fork is ready for PR,
if you have any questions or suggestions just let me know

**Comment by vn7n24fzkq** [comment]
Hi, @Cheezik.

Apologies for the delayed response.
I believe everything is ready now, as I've already added a wiki folder.
Could you please create a merge request and ensure that all the content is placed within the wiki folder?
This way, the wiki pages should be published from the wiki folder. Thank you for your assistance.

**Comment by vn7n24fzkq** [comment]
The PR #175 has been merged.
My apologies for the delay; I've been quite busy lately.
Everything looks fantastic, but it seems like there might be some broken links under the wiki homepage. https://github.com/vn7n24fzkq/github-profile-summary-cards/wiki

Would you mind taking a look? If you need any assistance with fixing them, please don't hesitate to let me know. Thank you for your attention to this matter!

**Comment by Cheezik** [comment]
Oh yeah.. i knew this might happen, i will possibly try to solve that today.

**Comment by vn7n24fzkq** [comment]
Hey, @Cheezik. Is this issue finished? If so, I'm going to close it. Thanks again for your hard work.

**Comment by Cheezik** [comment]
yeah im pretty sure the wiki is done, ill try to keep up and check on it if everything works as intended or update the info if anything changes. if there is something let me know ^^
